openmc/deplete/pool.py

```diff
@@ -15,8 +17,8 @@
 NUM_PROCESSES = None
 
 
-def deplete(func, chain, x, rates, dt, matrix_func=None):
-    """Deplete materials using given reaction rates for a specified time
+def deplete(func, chain, x, rates, dt, matrix_func=None, msr=None):
+    r"""Deplete materials using given reaction rates for a specified time
```

Comment:
Unless I'm missing something, this doesn't look like it needs to be a raw string
```suggestion
    """Deplete materials using given reaction rates for a specified time
```